src/forms/BoundCheckboxField.tsx

```diff
@@ -21,7 +22,7 @@ export function BoundCheckboxField(props: BoundCheckboxFieldProps) {
     onBlur,
     ...others
   } = props;
-  const testId = useTestIds(props, field.key);
+  const testId = useTestIds(props, defaultTestId(label));
```

Comment:
Doesn't UseTestIds do this defaulTestId() call?

Reply:
Ah yep!

Reply:
@bdow fixed!